LICENSE.md

```
@@ -3,6 +3,25 @@
 - **Core**: MIT (this file)
 - **Pro**: see [REACT-ON-RAILS-PRO-LICENSE.md](./REACT-ON-RAILS-PRO-LICENSE.md)
 
+## LicenseRef-Proprietary
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)